uxarray/_exodus.py

```diff
@@ -0,0 +1,223 @@
+            self.in_ds.Mesh2_node_y.data.tolist(),
+            self.in_ds.Mesh2_node_z.data.tolist()
+        ])
+
+    self.exo_ds["coord"] = xr.DataArray(data=c_data,
+                                        dims=["num_dim", "num_nodes"])
+
+    # passing the max node for element type
+    # might not work for mixed elements, same for reading
+    element_type = get_element_type(self.in_ds.nMaxMesh2_face_nodes.size)
+
+    self.exo_ds["connect1"] = xr.DataArray(
+        data=(self.in_ds.Mesh2_face_nodes[:] + 1),
+        dims=["num_el_in_blk1", "num_nod_per_el1"],
+        attrs={"elem_type": element_type})
+
+    self.exo_ds["edge_type1"] = xr.DataArray(data=xr.DataArray(
+        np.zeros(
+            (self.in_ds.nMesh2_face.size, self.in_ds.nMaxMesh2_face_nodes.size),
+            "i4")),
+                                             dims=[
+                                                 "num_el_in_blk1",
+                                                 "num_nod_per_el1"
+                                             ])
+
+    self.exo_ds["global_id1"] = xr.DataArray(data=(self.in_ds.nMesh2_face[:] +
+                                                   1),
+                                             dims=["num_el_in_blk1"])
+
+    # TODO: fix num attr
+    num_attr = 1
+    self.exo_ds["attrib1"] = xr.DataArray(
+        data=xr.DataArray(
+            np.zeros((self.in_ds.nMesh2_face.size, num_attr), float)),
+        dims=["num_el_in_blk1", "num_att_in_blk1"])
+
+    self.exo_ds["eb_prop1"] = xr.DataArray(data=xr.DataArray(np.ones((1),
+                                                                     "i4")),
+                                           dims=["num_el_blk"],
+                                           attrs={"name": "ID"})
+
+    self.exo_ds["eb_status"] = xr.DataArray(data=xr.DataArray(
+        np.array([num_attr], dtype="i4")),
+                                            dims=["num_el_blk"])
+
+    self.exo_ds["eb_names"] = xr.DataArray(data=xr.DataArray(
+        np.array([''], dtype="S33")),
+                                           dims=["num_el_blk"])
+
+    if dim == 2:
+        cnames = ["x", "y"]
+    elif dim == 3:
+        cnames = ["x", "y", "z"]
+
+    self.exo_ds["coor_names"] = xr.DataArray(data=xr.DataArray(
+        np.array(cnames, dtype="S33")),
+                                             dims=["num_dim"])
+
+    # done processing write the file to disk
+    self.exo_ds.to_netcdf(outfile)
+    print("Wrote: ", outfile)
+
+
+def get_element_type(num_nodes):
+    if num_nodes == 2:
+        element_type = "BEAM"
+    elif num_nodes == 3:
+        element_type = "TRI"
+    elif num_nodes == 4:
+        element_type = "SHELL4"
+    elif num_nodes == 8:
+        element_type = "SHELL8"
+    elif num_nodes == 9:
+        element_type = "SHELL9"
+    elif num_nodes == 6:
+        element_type = "TRI6"
+    elif num_nodes == 7:
+        element_type = "TRI7"
```

Comment:
please sort the elif tree by num_nodes.
there is no `elif num_nodes == 5:` which is a possible case.

Reply:
could consider making this a dict
``` python
ELEMENT_TYPE_DICT = {
    2: "BEAM",
    3: "TRI",
...
}
...
element_type = ELEMENT_TYPE_DICT[num_nodes]
````